<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matrixrecher</class>
 <widget class="QWidget" name="Matrixrecher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>371</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Matrixrechner</string>
  </property>
  <widget class="QPushButton" name="randomButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Zufalls-generator</string>
   </property>
  </widget>
  <widget class="QLabel" name="tauschLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>57</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Zeilentausch</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zeile</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tauschZeileButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>go</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="additionFaktor">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>* Zeile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>zu Zeile</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="additionZeileButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>go</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>16</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="multiplikationFaktor">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="multiplikationButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>go</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>330</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>... Matrixrechner ...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="dialZeileIn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="pageStep">
    <number>3</number>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dialZeileOut">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="pageStep">
    <number>3</number>
   </property>
   <property name="sliderPosition">
    <number>1</number>
   </property>
   <property name="notchTarget">
    <double>4.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="console">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxZeilen">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSpalten">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>177</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zeilen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>177</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Spalten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Aufr√§umen</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="zeileTausch1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="zeileTausch2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="additionVorzeichen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="multiplikationZeile">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
  </widget>
  <widget class="QComboBox" name="multiplikationVorzeichen">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
